A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_IRSStatusFrozen.js

```diff
@@ -0,0 +1,45 @@
+class CDUIRSStatusFrozen {
+    static ShowPage(mcdu, index) {
+        mcdu.clearDisplay()
+        let currPos = new LatLong(SimVar.GetSimVarValue("GPS POSITION LAT", "degree latitude"),
+                                  SimVar.GetSimVarValue("GPS POSITION LON", "degree longitude")).toShortDegreeString();
+        let GROUNDSPEED = SimVar.GetSimVarValue("GPS GROUND SPEED", "Meters per second") || "0";
+        let THDG = SimVar.GetSimVarValue("GPS GROUND TRUE HEADING", "radians") || "000";
+        let TTRK = SimVar.GetSimVarValue("GPS GROUND MAGNETIC TRACK", "radians") || "000";
+        let MHDG = SimVar.GetSimVarValue("GPS GROUND TRUE TRACK", "radians") || "000";
+        let WIND_DIR = SimVar.GetSimVarValue("AMBIENT WIND DIRECTION", "Degrees") || "000";
```

Comment:
Getting ambient wind seems to cause the windindicator to jump around **a lot**. Perhaps implementation via aircraft wind is wiser? See the implementation of this on the ND in A32NX_NDInfo.js where simplane values are used instead of simvars.

Reply:
``` javascript
function getWindDirection() {
        var angle = SimVar.GetSimVarValue("AMBIENT WIND DIRECTION", "Degrees");
        return angle;
    }
    Simplane.getWindDirection = getWindDirection;
```
This is the Simplane function for getting the wind direction.
Simplane is most of the time (not always) just a wrapper for SimVar to make your life easier, it should be exactly the same in this case.

Reply:
Mh, I agree with you but I don't really understand where the discrepancy between the two values comes from then. The severity of this however seems to depend on the exact weather conditions, it's more noticable with stronger winds. 

https://streamable.com/0iozx7